lib/Models/RegionParameter.js

```diff
@@ -96,7 +96,11 @@ RegionParameter.prototype.findRegionByID = function(regionID) {
     }
 
     return regionProvider.regions.filter(function(region) {
-        return region.id === regionID;
+        if (typeof region.id === 'string' && typeof regionID === 'string') {
```

Comment:
This is the change that allows countries to be selected by clicking on them. Otherwise you get to here and (for example) Australia is compared to australia and the region id is not returned. Do I need to fix this in regionProvider instead though?